App/Logic/ExposureDetectionLogic.swift

```diff
@@ -44,6 +44,8 @@ extension Logic.ExposureDetection {
       }
 
       let state = context.getState()
+      let countriesOfInterest = state.exposureDetection.countriesOfInterest
```

Comment:
As written above, I would use a struct here to avoid issues related to use an array of tuples